agents/notion/main.py

```diff
@@ -0,0 +1,73 @@
+# This is a Fixie Agent that pulls a list of URLs from a Notion database and answers
+# questions about them. See README.md for details on how to set this up for yourself.
+
+import os
+import fixieai
+
+from notion_client import Client
+
+# The Notion Database to pull the list of URLs from. This database must be shared with your
+# Notion integration - see README.md for details.
+NOTION_DATABASE_ID = "b264de3998384f839245bd54faa40d9c"
+
+BASE_PROMPT = """I am an agent that answers questions about a set of web pages, stored in a Notion database."""
+
+FEW_SHOTS = """
```

Comment:
I don't think we need these few_shots, in fact I am going to remove few-shots from all docs agents as they just waste LLM inferences (in favor of a new pattern that I am working on). Just leave one or two for now showing the query_corpus pattern.